hack/jenkins/kicbase_auto_build.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+
+# Copyright 2021 The Kubernetes Authors All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+set -x
+
+# Make sure docker is installed and configured
+./hack/jenkins/installers/check_install_docker.sh
+yes|gcloud auth configure-docker
+
+# Setup variables
+now=$(date +%s)
+KV=$(egrep "Version =" pkg/drivers/kic/types.go | cut -d \" -f 2 | cut -d "-" -f 1)
+KIC_REPO=gcr.io/k8s-minikube/kicbase-builds
+export KIC_VERSION=$KV-$now-$ghprbPullId
+export KICBASE_IMAGE_REGISTRIES=$KIC_REPO:$KIC_VERSION
+
+# Let's make sure we have the newest kicbase reference
+curl -L https://github.com/kubernetes/minikube/raw/master/pkg/drivers/kic/types.go --output types-head.go
+HEAD_KIC_TIMESTAMP=$(egrep "Version =" types-head.go | cut -d \" -f 2 | cut -d "-" -f 2)
```

Comment:
comment here the evaluated $() as example what should be expected in the HEAD_KIC_TIMESTAMP
, will this still work if the HEAD is a stable kic image without the PR number?

Reply:
yeah, cut is a very safe operation to use

Reply:
HEAD_KIC_TIMESTAMP seems to be a pretty clear variable name, the comment would just read `# HEAD_KIC_TIMESTAMP is the timestamp of the kicbase build at HEAD.`

Reply:
the reason for this, when there is no timestamp (on a released kicbase) the nextperson would need to debug what this expects to see

so the comment could be ...

/// line "...." grep "...."

Reply:
if there is no timestamp, then cut will just return the full version, e.g. v0.0.17, which is covered by the if statement below

Reply:
i added a comment for clarity